app/views/albums/index.html.erb

```diff
@@ -0,0 +1,46 @@
+<div class="albums__topNav">
+    <%= button_to 'ADD NEW', new_album_path, { method: 'get', class: "albums__addButton" } %>
+</div>
+
+<table class="albums__table">
+  <tr class="albums__tableHeader">
+    <th><h3>Title</h3></th>
+    <th><h3>Artist</h3></th>
+    <th><h3>Details</h3></th>
+    <th><h3>Delete</h3></th>
+  </tr>
+  <% if @albums.length > 0 %>
+    <% @albums.each_with_index do |album, i| %>
+      <tr class=
+        <% if i % 2 == 0 %>
+            <%= "albums__tableRow" %>
+          <% else %>
+            <%= "albums__tableRowAlt" %>
+          <% end %>
+        >
+        <th>
+          <%= album.title %>
+        </th>
+        <th>
+          <%= album.artist %>
+        </th>
+        <th>
+          <%= button_to 'MORE >>', album_path(album.id), method: 'get' %>
+        </th>
+        <th>
+          <%= button_to('DELETE', album_path(album.id), method: 'delete', remote: false, data: { confirm: 'Are you sure?', disable_with: 'loading...'}) %>
```

Comment:
I'm saying I didn't know that was a thing. Always be learning. 



—
Robert Pearce

http://www.robertwpearce.com

https://www.linkedin.com/in/robertwpearce



m-US: 1.843.568.9080

m-UK: +44 7477 424374

On Sun, Jul 12, 2015 at 5:16 PM, Andrew T. Poe <notifications@github.com>
wrote:

>> +            <%= "albums__tableRow" %>
>> +          <% else %>
>> +            <%= "albums__tableRowAlt" %>
>> +          <% end %>
>> +        >
>> +        <th>
>> +          <%= album.title %>
>> +        </th>
>> +        <th>
>> +          <%= album.artist %>
>> +        </th>
>> +        <th>
>> +          <%= button_to 'MORE >>', album_path(album.id), method: 'get' %>
>> +        </th>
>> +        <th>
>> +          <%= button_to('DELETE', album_path(album.id), method: 'delete', remote: false, data: { confirm: 'Are you sure?', disable_with: 'loading...'}) %>
> Why no disable_with?
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/andrewtpoe/4.4-CRUD/pull/1/files#r34425700